### ISSUE TITLE
Specifying setup.py encoding formats

### ISSUE THREAD

**Issue by akaiuun12** [jsvine/pdfplumber#319]
## Describe the bug
Installation using pip was failed with 'cp949 error'. The error occurs only to the Windows with specific language using cp949 format, Korean in my case. No problem on Mac, not tested on Linux. 

This error is not specific to pdfplumber and can be found quite frequently. While Korean users can solve the problem individually, specifying the encoding formats of the setup file can simply fix the issue.

## Expected behavior
![image](https://user-images.githubusercontent.com/64883793/100529965-c2eb8a00-322f-11eb-9dbb-1fa6a905fb34.png)

## Actual behavior
![image](https://user-images.githubusercontent.com/64883793/100529944-92a3eb80-322f-11eb-895b-3fdbe99fccc4.png)

```bash
UnicodeDecodeError: 'cp949' codec can't decode byte 0xe2 in position 4981: illegal multibyte sequence
```

## Environment

- pdfplumber version: 0.5.24
- Python version: 3.8.5
- OS: Windows 10

## Additional context

As noted above, the error is because of the unspecified encoding formats of the setup file in a certain Windows environment. The problem can be easily fixed by adding the encoding parameter to the setup file as below. I'd willing to send a merge request if the project manager is willing to fix this bug. Thanks.

```
with open(os.path.join(HERE, "README.md"), encoding='UTF-8') as f:
    long_description = f.read()
```

**Comment by jsvine** [comment]
Thank you for the detailed and well-described report, @akaiuun12! Thankfully, this has already been fixed in the `develop` branch via https://github.com/jsvine/pdfplumber/commit/78543284187586f8ca8428a73f93efb714152693 and will be available via PyPI in the next release.

**Comment by jsvine** [comment]
Now available in v0.5.25.
